commands/command_merge_driver.go

```diff
@@ -0,0 +1,140 @@
+package commands
+
+import (
+	"fmt"
+	"os"
+	"os/exec"
+
+	"github.com/git-lfs/git-lfs/v3/errors"
+	"github.com/git-lfs/git-lfs/v3/lfs"
+	"github.com/git-lfs/git-lfs/v3/subprocess"
+	"github.com/git-lfs/git-lfs/v3/tr"
+	"github.com/spf13/cobra"
+)
+
+var (
+	mergeDriverAncestor   string
+	mergeDriverCurrent    string
+	mergeDriverOther      string
+	mergeDriverOutput     string
+	mergeDriverProgram    string
+	mergeDriverMarkerSize int
+)
+
+func mergeDriverCommand(cmd *cobra.Command, args []string) {
+	if len(mergeDriverAncestor) == 0 || len(mergeDriverCurrent) == 0 || len(mergeDriverOther) == 0 || len(mergeDriverOutput) == 0 {
+		Exit(tr.Tr.Get("the --ancestor, --current, --other, and --output options are mandatory"))
```

Comment:
I'd rather not do so, because we have four mandatory arguments and once you have more than three positional arguments (on the command line or in a function), it gets tricky to remember what order they go in, especially when they're all of similar types.

Reply:
OK ... I don't love that myself, as I feel like options are, well, always optional and not mandatory arguments, plus there's the divergence from Git's example.  But I can live with it if it's well documented.